frontend/app/src/main/res/layout/activity_create_new_list.xml

```diff
@@ -4,4 +4,99 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".CreateNewListActivity"/>
\ No newline at end of file
+    tools:context=".CreateNewListActivity">
+
+
+    <TextView
+        android:id="@+id/create_list_title_prompt"
+        android:layout_width="346dp"
+        android:layout_height="wrap_content"
+        android:gravity="center"
+        android:text="@string/create_list_title_prompt_text"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.04000002" />
+
+    <TextView
+        android:id="@+id/create_list_name"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/create_list_name_prompt"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/create_list_title_prompt"
+        app:layout_constraintVertical_bias="0.050000012" />
+
+    <EditText
+        android:id="@+id/create_list_name_exit_text"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="506dp"
+        android:hint="@string/create_list_name_edit_text_string"
+        android:inputType="text"
+        android:autofillHints="Name"
+    app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/create_list_name" />
+
+    <TextView
+        android:id="@+id/create_list_description_name"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/create_list_description_name_prompt"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/create_list_title_prompt"
+        app:layout_constraintVertical_bias="0.18" />
+
+    <EditText
+        android:id="@+id/create_list_description_edit_text"
+        android:layout_width="match_parent"
```

Comment:
I think match_parent makes it go all the way to the edge. 
May want to use some kind of minimum and a maximum